fastlane/Fastfile

```diff
@@ -269,41 +269,67 @@ platform :ios do
   end
 
   #####################################################################################
-  # finalize_release
+  # check_all_translations
   # -----------------------------------------------------------------------------------
-  # This lane finalize a release: updates store metadata, bump final version number,
-  # remove branch protection and close milestone, then trigger the final release on CI
+  # This lane checks the translation progress for all the projects in GlotPress.
+  #
+  # By default, it will print all values. You can make it ask whether to continue when
+  # if found translations that are below the threshold by passing `interactive: true`.
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane finalize_release [skip_confirm:<skip confirm>] [version:<version>]
+  # bundle exec fastlane check_all_translations [interactive:<interactive>]
   #
   # Example:
-  # bundle exec fastlane finalize_release
-  # bundle exec fastlane finalize_release skip_confirm:true
+  # bundle exec fastlane check_all_translations
+  # bundle exec fastlane check_all_translations interactive:true
   #####################################################################################
-  desc 'Trigger the final release build on CI'
-  lane :finalize_release do |options|
-    UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
+  desc 'Check translation progress for all GlotPress projects'
+  lane :check_all_translations do |options|
```

Comment:
The thought occurred to me to make this a bit more organized so that one could check the individual progress.

At this time, I don't see the need for checking individual projects. So it would be a "nice to have" that would bring unnecessary overhead. Besides, at that point, it wouldn't be much different than calling `check_translation_progress` manually, just with less typing.